### ISSUE TITLE
iOS: Feeds list - background behind table view should be white

### ISSUE THREAD

**Issue by brentsimmons** [brentsimmons/NetNewsWire#1224]
System background color, which is white, or dark in Dark Mode.

Also, and related — the background for the big Feeds text should be white/dark.

The Feeds list table view should use plain style rather than grouped style.

**Comment by vincode-io** [comment]
I made the change for the background color.

Regarding plain style vs grouped style, switching to plain style is very, very difficult unless you want to switch to floating section headers.  I've tried several times to switch to plain style and still not have floating section headers and all the hacks to do it have unacceptable side effects.

The other option to use plain style is to not use section headers at all and emulate them using regular table cells.  This would require me to change the structure of the shadow backing table I use for the table view.  This is a huge change that would impact much more than just the layout of the table.  It impacts code like finding the next unread article and more.  Just thinking about doing this kind of surgery this late in the project makes me queasy.

I *think* from previous conversations that what you are asking for is grouped style behavior with plain style appearance.  I've tried to do this by using the grouped style and emulating the plain style.  I've stuck to this strategy for now and updated the latest release to more closely emulate plain style.

If I've missed the mark here and there is something about the grouped style behavior that you don't like, let me know.  Alternately, if I haven't correctly emulated plain style, I can correct that.  Or, if you want plain style with floating headers, I can do that too.  I just don't think that plain style without floating headers is doable right now.
